vm/systemSmartContracts/esdt.go

```diff
@@ -374,6 +388,13 @@ func (e *esdt) registerSemiFungible(args *vmcommon.ContractCallInput) vmcommon.R
 
 	e.eei.Finish(tokenIdentifier)
 
+	logEntry := &vmcommon.LogEntry{
+		Identifier: []byte(args.Function),
+		Address:    args.CallerAddr,
+		Topics:     [][]byte{tokenIdentifier, args.Arguments[0], args.Arguments[1], []byte(core.SemiFungibleESDT)},
+	}
+	e.eei.AddLogEntry(logEntry)
```

Comment:
you have to do the same for MetaESDT as well. And for change into metaESDT as well.

Reply:
done.